examples/models/llama/install_requirements.sh

```diff
@@ -17,6 +17,8 @@ pip install --no-use-pep517 "git+https://github.com/pytorch/ao.git@${TORCHAO_VER
 # Install tiktoken for tokenizer
 pip install lm_eval==0.4.2
 pip install tiktoken blobfile
+# Restore numpy if >= 2.0
+pip install "numpy<2.0"
```

Comment:
what is this change for?

Reply:
Oh, running this script will mess up the numpy version though CI complains it only as a warning. It's to restore to the compatible numpy version as specified in pyproject.toml: https://github.com/pytorch/executorch/blob/main/pyproject.toml#L58-L60